README.md

```diff
@@ -27,6 +27,28 @@ Set a destination folder for moving files with the `f` flag. The folder will be
 
 ```$ riv -f ~/saved_images```
 
+Set a sorting order with the `s` or `--sort` flag, case insensitive.
+
+```$ riv -s alphabetical **/*.png```
+
+### Sorting Options
```

Comment:
Good idea to explain the sorting options.

I do wonder if Clap and/or a man page generator has an option to express these details without looking at the project readme..